tests/unit/downloader/test_xls_generation.py

```diff
@@ -0,0 +1,165 @@
+                    "project.insdc_study_accessions"
+                ],
+                "values": [
+                    {
+                        "project.uuid": "3e329187-a9c4-48ec-90e3-cc45f7c2311c",
+                        "project.project_core.project_short_name": "kriegsteinBrainOrganoids",
+                        "project.project_core.project_title": "Establishing Cerebral Organoids as Models of Human-Specific Brain Evolution",
+                        "project.project_core.project_description": "Direct comparisons of human and non-human primate brain tissue have the potential to reveal molecular pathways underlying remarkable specializations of the human brain. However, chimpanzee tissue is largely inaccessible during neocortical neurogenesis when differences in brain size first appear. To identify human-specific features of cortical development, we leveraged recent innovations that permit generating pluripotent stem cell-derived cerebral organoids from chimpanzee. First, we systematically evaluated the fidelity of organoid models to primary human and macaque cortex, finding organoid models preserve gene regulatory networks related to cell types and developmental processes but exhibit increased metabolic stress. Second, we identified 261 genes differentially expressed in human compared to chimpanzee organoids and macaque cortex. Many of these genes overlap with human-specific segmental duplications and a subset suggest increased PI3K/AKT/mTOR activation in human outer radial glia. Together, our findings establish a platform for systematic analysis of molecular changes contributing to human brain development and evolution. Overall design: Single cell mRNA sequencing of iPS-derived neural and glial progenitor cells using the Fluidigm C1 system This series includes re-analysis of publicly available data in accessions: phs000989.v3, GSE99951, GSE86207, GSE75140. Sample metadata and accession IDs for the re-analyzed samples are included in the file \"GSE124299_metadata_on_processed_samples.xlsx\" available on the foot of this record. The following samples have no raw data due to data loss: GSM3569728, GSM3569738, GSM3571601, GSM3571606, GSM3571615, GSM3571621, GSM3571625, and GSM3571631",
+                        "project.insdc_project_accessions": "SRP180337",
+                        "project.geo_series_accessions": "GSE124299",
+                        "project.insdc_study_accessions": "PRJNA515930"
+                    }
+                ]
+            }
+        }
+
+        # when
+        workbook: Workbook = self.downloader.create_workbook(input_json)
+
+        #expect
+        self.assertEqual(len(workbook.worksheets), 1)
+
+        self.__assert_sheet(workbook, project_sheet_title, input_json)
+
+    def test_given_input_has_many_rows_of_data_successfully_creates_a_workbook(self):
+        #given
+        contributors_sheet_title = 'Project - Contributors'
+
+        input_json = {
+            contributors_sheet_title: {
```

Comment:
Is it possible to move this to a spec file? It makes the tests a bit hard to read